collectors/proc.plugin/plugin_proc.c

```diff
@@ -98,6 +98,37 @@ static void proc_main_cleanup(void *ptr)
     worker_unregister();
 }
 
+bool inside_lxc_container = false;
+
+static bool is_lxcfs_proc_mounted() {
```

Comment:
We are calling this function before to start main loop,  but we are defining `ff` as `static` and we are not closing it, I suggest to avoid memory leak the following modifications:

1. Remove `static` for cited variable.
2. If we have success to open file, before to return call `procfile_close`. We have an example [here](https://github.com/netdata/netdata/blob/cbbd338cad8848064a72a579c152e56a3975b96b/collectors/apps.plugin/apps_plugin.c#L616-L666).

Except for this small improvement, the PR is running as expected.